_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -1,39 +1,45 @@
 ---
-name: Heading is descriptive
+id: 5e4949
+name: Visible heading is descriptive
 rule_type: atomic
 
 description: |
-   This rule checks that headings describe the topic or purpose of the content.
+   This rule checks that each visible heading describe the topic or purpose of the content.
    
-success_criterion:
-- 2.4.6 # Headings and labels
-
-test_aspects:
+accessibility_requirements:
+- wcag20:2.4.6 # Headings and labels (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
 - DOM Tree
 - CSS Styling
 
 authors:
+- Anne Thyme Nørregaard
 - Dagfinn Rømen
 - Geir Sindre Fossøy
 - Carlos Duarte
 ---
 
-## Test procedure
+## Applicability
 
-### Applicability
+This rule applies to any [visible](#visible) element with the [semantic role](#semantic-role) of `heading`.
 
-This rule applies to any element with the [semantic role](#semantic-role) of heading that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+**Note:**** Headings with only [whitespace](#whitespace) are not [visible](#visible).
 
-**Note**: This rule only applies to elements with the [semantic role](#semantic-role) of heading. Thus, it is a partial check for WCAG 2.0 success criterion 2.4.6, which applies to all headings. "Heading" is used in its general sense and includes headlines and other ways to add a heading to different types of content. This includes elements that are not marked up as headings in the code, but still act visually as headings, e.g. by larger and/or bolder text. 
+## Expectation
 
-### Expectation
+The [visible](#visible) heading of each target element describes the topic or purpose of its [visible](#visible) [section of the content](#section-of-content) in part or in its entirety, or the `heading` has no [visible](#visible) [section of the content](#section-of-content).
 
-Each target element describes the topic or purpose of its [section of the content](#section-of-content).
+The [visible](#visible) heading is the combined [characters](#character) formed by the [visible](#visible) [descendant text content](#) and [characters](#character) in [images of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text).
```

Comment:
Remove link to character and insert note about definition missing + create issue on character definition.